applications/Cyborg.md

```diff
@@ -0,0 +1,178 @@
+- https://github.com/ashi20192020
+- https://github.com/dheeraj07
+- https://github.com/Megha2712
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/barath-kanna-23a23a172 
+- https://www.linkedin.com/in/kresna-sucandra
+- https://www.linkedin.com/in/megha-varshini-tamilarasan-b1247a212
+
+## Development Status :open_book:
+
+We initially developed a Rust WebSocket client with a low-key profile, ensuring a seamless connection with our blockchain through off-chain workers. Additionally, we created a proof of concept based on this client. Our initial plan was to proceed with this development. However, based on the recommendation from the W3F grants team during our [previous application](https://github.com/w3f/Grants-Program/pull/1933) to eliminate centralized entities from our architecture and establish a fully decentralized system from the ground up, we embarked on several research iterations.
+
+Through this process, we identified the potential of using connected clusters than indivdual instances for better management efficiency, We are currently building an on chain work verifcation system. Our team will shift focus to building custom pallets outlined in the technical architecture to steer our way through to become a parachain on Kusama and eventually on Polkadot.
+
+As part of our transition, we are migrating from WebSockets to utilizing Libp2p as the core connection protocol. Furthermore, Subxt will serve as the standard communication protocol facilitating interaction between the Substrate chain and the off-chain domain. This strategic shift aligns with our commitment to adopting decentralized principles and enhancing the robustness of our system architecture.
+
+We have created a UX demo for a potential user and the development for this grant will look very similar to this [Cyborg Network UX demo](https://drive.google.com/file/d/1zrN4TRMDd5ON2xH9F00upcyK-3ofwV4r/view?usp=drivesdk)
+
+
+## Development Roadmap :nut_and_bolt:
+
+We will be developing a PoC as a part for this grant.
+
+The goal is to achieve a functional system that is capable of scheduling and verifying a batch process (Shell Script or Docker container) that was executed in an off - chain entity.
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 2 ( 1 Full time Developers at 1 FTE/month and 2 part time developers each at 0.5 FTE/month)
+- **Total Costs:** 10,000 USD
+
+### Milestone 1 - Task Verification System
+
+- **Estimated duration:** 2 months
+- **FTE:**  4
+- **Costs:** 10,000 USD
+
+#### **Deliverables:**
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | GPLv3 |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
```

Comment:
please specify the documentation delivery so it fits your actual product.

Reply:
Done, Thanks